code-examples/sdk/typescript/src/session/refresh-session-native.ts

```diff
@@ -0,0 +1,16 @@
+import { Configuration, IdentityApi } from "@ory/client"
+
+const identityApi = new IdentityApi(
+  new Configuration({
+    basePath: `https://${process.env.ORY_PROJECT_SLUG}.projects.oryapis.com`,
+    apiKey: `${process.env.ORY_ACCESS_TOKEN}`,
```

Comment:
`apiKey` works here? Interesting :thinking: 
So this would automatically add the header `Authorization`: `Bearer ${token}` ?

Reply:
correction - this should be `accessToken`. Thanks!

Reply:
fixed https://github.com/ory/docs/pull/1185/commits/cf9380510ad32faad8b2af0142a5ca2eaf9fe119